services/api/src/models/group.ts

```diff
@@ -53,38 +62,6 @@ interface AttributeFilterFn {
   (attribute: { name: string; value: string[] }, group: Group): boolean;
 }
 
-interface GroupModel {
```

Comment:
Was this redundant? I used it when creating the model to figure out what the API should look like and get autocomplete working

Reply:
Yeah, if you ever need to get the type of the GroupModel, you can do: 

```
ReturnType<typeof GroupModel>
```

This will return whatever is defined in the function

Reply:
I do that here in fact
https://github.com/justinlevi/lagoon/blob/dfade1133717f6bd1125dfa39416bf637c2c9a70/services/api/src/resources/billing/helpers.ts#L30